packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js

```diff
@@ -6,13 +6,17 @@ import {
 
 // keyValValidator takes as a parameter cmcdJson, which is a javascript object.
 // The function iterates through it validating every key value pair.
-const keyValValidator = (cmcdJson, errors, warnings, warningFlag = true) => {
+const keyValValidator = (cmcdJson, errors, warnings, config, warningFlag = true) => {
 
   if (warningFlag === true) {
     checkSidIsPresent(cmcdJson, warnings);
   }
 
   Object.keys(cmcdJson).forEach((key) => {
+    // Chech if we recived a configuration
+    if (config?.specificKey && !config.specificKey?.includes(key)) {
```

Comment:
Is necessary to check config?.specificKey?

Reply:
yes it is, if we don't check it we can't check the second condition